### PR TITLE
Add overriding metaclass.

### PR DESCRIPTION
**Pull request by hameerabbasi**
cc @joaosferreira

### CODE REVIEW

**Comment by peterbell10** [comment]
You seem to have commit a binary file by accident. I assume `.profraw` is some profiler output?

**Comment by hameerabbasi** [comment]
> I assume `.profraw` is some profiler output?

It shows up with a simple `pytest`, I'm not entirely clear on why.

File: unumpy/numpy_backend.py [hunk 1]
```diff
@@ -11,30 +11,44 @@
 __ua_domain__ = "numpy"
 
 
+def overridden_class(self):
```
Comment (hameerabbasi) [inline unumpy/numpy_backend.py:14]: @joaosferreira For Dask, this would have an additional:
```
if self is np.ndarray:
    return da.Array
```

**Comment by hameerabbasi** [comment]
Superceded by #66.

**Pull request closed without merging**